specifyweb/frontend/js_src/lib/components/Notifications/hooks.tsx

```diff
@@ -10,7 +10,7 @@ import type { GenericNotification } from './NotificationRenderers';
 
 const INITIAL_INTERVAL = 5000;
 const INTERVAL_MULTIPLIER = 1.1;
-let addSinceToNotification = true;
+// let addSinceToNotification = true;
```

Comment:
from @maxxxxxdlp 
this is a wrong location. it should be as a ref inside of useNotificationsFetch()
the notifications state is initialized to undefined when the hook is first called - so this boolean flag should be initialized at the same time

the Notifications component may be re-rendered completely (maybe when user changes the menu position)

Reply:
line 85:  lastFetchedTimestamp = startFetchTimestamp;
this should probably also be stored in a ref - right now lastFetchedTimestamp value is lost on useEffect re-executions (happens every time the user opens/closes the notifications component)